cartographer/sensor/proto/sensor.proto

```diff
@@ -44,3 +44,10 @@ message FixedFramePoseData {
   optional int64 timestamp = 1;
   optional transform.proto.Rigid3d pose = 2;
 }
+
+// Proto representation of ::cartographer::sensor::LandmarkData.
+message LandmarkData {
+  optional string id = 1;
```

Comment:
Should this not include a `timestamp` similar to `FixedFramePoseData`?

We have to be able to report the detection of multiple or no landmarks, e.g. if we detect no landmarks we still have to provide sensor data to allow the SLAM algorithm to advance. Use a `repeated` field of observation message?